examples/street_canyon_radiosity.ipynb

```diff
@@ -0,0 +1,509 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "brdf_in = sph.samplings.gaussian(n_max=15).to_pyfar()  # 32x16 / 2 points\n",
+    "brdf_in = brdf_in[brdf_in.z > 0]\n",
+    "brdf_out = brdf_in.copy()\n",
+    "\n",
+    "bsc_retro_reflection = np.ones(\n",
+    "    (brdf_in.csize, brdf_out.csize, len(frequencies)),\n",
+    ")\n",
+    "bsc_retro_reflection /= 2 * brdf_out.csize\n",
+    "bsc_retro_reflection[\n",
+    "    np.arange(brdf_in.csize),\n",
+    "    np.arange(brdf_out.csize),\n",
+    "    0,\n",
+    "] += 0.5\n",
+    "\n",
+    "bsc_retro_reflection = pf.FrequencyData(bsc_retro_reflection, frequencies)\n",
+    "absorption_coefficient = 0.1\n",
+    "absorption_data = pf.FrequencyData(\n",
+    "    [absorption_coefficient],\n",
+    "    frequencies,\n",
+    ")\n",
+    "\n",
+    "brdf_retro = sp.brdf.create_from_directional_scattering(\n",
+    "    brdf_in,\n",
+    "    brdf_out,\n",
+    "    bsc_retro_reflection,\n",
+    "    absorption_coefficient=absorption_data,\n",
+    ")\n",
+    "\n",
+    "\n",
+    "radiosity_retroreflection.set_wall_brdf(\n",
+    "    np.arange(len(boundaries)).tolist(),\n",
+    "    brdf_retro,\n",
+    "    brdf_in,\n",
+    "    brdf_out,\n",
+    ")\n",
+    "\n",
+    "# Plotting BRDF data\n",
+    "fig = plt.figure()\n",
+    "ax = fig.add_subplot(projection=\"3d\")\n",
+    "ax.scatter(\n",
+    "    brdf_out.x,\n",
+    "    brdf_out.y,\n",
+    "    brdf_out.z,\n",
+    ")\n",
+    "plt.title(\"BRDF sampling points\")\n",
+    "plt.show()"
```

Comment:
not sure it we really need it, if you want to keep it, use [pyfar.plot.scatter](https://pyfar--872.org.readthedocs.build/en/872/modules/pyfar.plot.html#pyfar.plot.scatter) instead